torch/onnx/experimental/fonnx/export_options.py

```diff
@@ -0,0 +1,54 @@
+class ExportStrategy(Enum):
```

Comment:
Remove this from pass 1. We can discuss how to handle fallback and its interaction with the exporter in the context of the registry API.